api-admin/src/main/java/bio/terra/pearl/api/admin/service/kit/KitExtService.java

```diff
@@ -47,21 +47,19 @@ public KitRequestListResponse requestKits(
       try {
         KitRequest kitRequest = requestKit(adminUser, studyShortcode, enrolleeShortcode, kitType);
         response.kitRequests.add(kitRequest);
-      } catch (PepperApiException pepperApiException) {
-        // add the enrollee shortcode to the message for disambiguation
-        response.pepperApiExceptions.add(
-            new PepperApiException(
-                enrolleeShortcode + ": " + pepperApiException.getMessage(),
-                pepperApiException.getErrorResponse(),
-                pepperApiException.getHttpStatus()));
+      } catch (Exception e) {
```

Comment:
You could keep the `PepperApiException` processing and add `Exception` processing since that seems useful?

Reply:
good point.  we'll see how this shakes out once we have a better overall exception-handling strategy.